### ISSUE TITLE
Start-date property does not update date component of moment-picker that only has time format specified

### ISSUE THREAD

**Issue by CareerFairPlus** [indrimuska/angular-moment-picker#261]
Setting the start-date property of a moment-picker should update its date component even if the format only specifies a time component.

See [this Plunker example](https://embed.plnkr.co/Bs0JRxX0hldYeHWpTXUg/)

If `ctrl.startDate` is set to 09/01/2018, this moment-picker still shows today's date at the top of the picker when choosing the time.

```
<input class="form-control"
             placeholder="Select a start time..."
             moment-picker
             locale="en"
             format="h:mm a"
             start-date="ctrl.startDate"
             ng-model="ctrl.startTime">
```

<img width="231" alt="image" src="https://user-images.githubusercontent.com/17342716/41922873-13ff74b8-7934-11e8-91a5-79f152472455.png">

**Comment by pratikp91** [comment]
@CareerFairPlus did u find the fix for this?Facing the same issue.Maybe you could help me?

**Comment by CareerFairPlus** [comment]
@pratikp91 No, never could get this working.  I chose to combine the date and time into a single input to get around this issue.  Still waiting for a real solution.
